definitions/tools/picard_merge_vcfs.cwl

```diff
@@ -9,7 +9,7 @@ requirements:
       ramMin: 40000
     - class: InlineJavascriptRequirement
     - class: DockerRequirement
-      dockerPull: "mgibio/picard-cwl:2.18.1"
+      dockerPull: "broadinstitute/picard:2.23.6"
```

Comment:
Sorry about creating a conflict in your PR!  I changed this tool in the Varscan scattering one.  Probably could just leave this one out.